### PR TITLE
[onert] Support type-aware quant for nnpkg io

### PR DESCRIPTION
**Pull request by ragmani**
This commit apply type-aware quantization to nnpkg io.

ONE-DCO-1.0-Signed-off-by: ragmani <ragmani0216@gmail.com>

---
For #10002
Draft #10329

### CODE REVIEW

File: runtime/onert/core/src/exec/Executors.cc [hunk 1]
```diff
@@ -149,9 +149,21 @@ void Executors::checkSupportedMultimodel() const
...
+  {
+    if (std::find(_model_edges->pkg_inputs.begin(), _model_edges->pkg_inputs.end(), edge.to) !=
+          _model_edges->pkg_inputs.end() ||
+        std::find(_model_edges->pkg_outputs.begin(), _model_edges->pkg_outputs.end(), edge.from) !=
+          _model_edges->pkg_outputs.end())
+    {
+      throw std::runtime_error{
+        "Multi model execution does not support duplicating nnpkg inputs/outputs with edges yet"};
+    }
+  }
```
Comment (hseok-oh) [inline runtime/onert/core/src/exec/Executors.cc:164]: Is it possible case when edge's end is nnpkg's input? (maybe possible edge's begin is pkg's out)

Comment (ragmani) [reply runtime/onert/core/src/exec/Executors.cc:164]: Does `edge's end` you mentioned means `to` in an edge?

Comment (ragmani) [reply runtime/onert/core/src/exec/Executors.cc:164]: Did you mean that `from` in edge could be an output of nnpkg?
It's possible.

| possibility of duplication | Edge `to` (input of subgraph) | Edge `from` (output of subgraph) |
| :---: | :---: | :---: |
| nnpkg input (input of subgraph) | X (*1) | X (*2) |
| nnpkg output (output of subgraph) | X (*2) | O |

*1. The subjects who determine values of each buffer are different.
  - nnpkg input : user input
  - Edge `to` : output of another subgraph

*2. `IOIndex` of inputs and outputs of subgraph is distinct.

Comment (hseok-oh) [reply runtime/onert/core/src/exec/Executors.cc:164]: My comment is about `std::find(_model_edges->pkg_inputs.begin(), _model_edges->pkg_inputs.end(), edge.to) != _model_edges->pkg_inputs.end()`.

Comment (hseok-oh) [reply runtime/onert/core/src/exec/Executors.cc:164]: My comment is about `std::find(_model_edges->pkg_inputs.begin(), _model_edges->pkg_inputs.end(), edge.to)`.

Comment (ragmani) [reply runtime/onert/core/src/exec/Executors.cc:164]: > My comment is about std::find(_model_edges->pkg_inputs.begin(), _model_edges->pkg_inputs.end(), edge.to). I thought it's impossible case.

I see. I will separate impossible cases from possible cases and throw those two exceptions separately.

Comment (hseok-oh) [reply runtime/onert/core/src/exec/Executors.cc:164]: Impossible cases should check on package loading phase, IMHO.

**Review by hseok-oh — APPROVED** [review]

**Review by jyoungyun — APPROVED** [review]
LGTM

**Pull request merged**